app/controllers/v1/interventions_controller.rb

```diff
@@ -73,12 +74,19 @@ def generated_conversations_transcript
 
   def interventions_scope
     @interventions_scope ||= Intervention.accessible_by(current_ability)
-                                         .order(created_at: :desc)
                                          .includes(%i[user reports_attachments files_attachments google_language logo_attachment logo_blob collaborators
                                                       conversations_transcript_attachment])
                                          .only_visible
   end
 
+  def sorted_interventions_scope
+    @sorted_interventions_scope ||= Intervention.accessible_by(current_ability)
+                                                .joins("LEFT JOIN (SELECT * FROM stars WHERE user_id = '#{current_v1_user.id}') AS user_stars
+                                                        ON user_stars.intervention_id = interventions.id")
+                                                .order('user_stars.user_id', 'interventions.created_at DESC')
+                                                .only_visible
```

Comment:
I think we shouldn't omit the includes, especially when we return a bunch of objects